### PR TITLE
Allow usage of pathlib.Path in VoxelData.save_nrrd

### PR DESCRIPTION
**Pull request by eleftherioszisis**

### CODE REVIEW

File: voxcell/voxel_data.py [hunk 1]
```diff
@@ -128,7 +128,7 @@ def save_nrrd(self, nrrd_path, encoding=None):
         """Save a VoxelData to an nrrd file.
 
         Args:
-            nrrd_path(string): full path to nrrd file
+            nrrd_path(string|pathlib.Path): full path to nrrd file
```
Comment (GianlucaFicarelli) [inline voxcell/voxel_data.py:131]: For completeness it's possible to apply the same changes in `load_nrrd`

File: tests/test_voxel_data.py [hunk 2]
```diff
@@ -100,6 +101,21 @@ def test_save_nrrd():
         assert np.allclose(vd.raw, new.raw)
 
 
+def test_save_nrrd_load_nrrd_str_Path():
+    """Test saving and loading as string and pathlib.Path"""
+    path = os.path.join(DATA_PATH, 'scalar.nrrd')
```
Comment (GianlucaFicarelli) [inline tests/test_voxel_data.py:106]: This path is unused.

File: tests/test_voxel_data.py [hunk 3]
```diff
@@ -100,6 +101,21 @@ def test_save_nrrd():
         assert np.allclose(vd.raw, new.raw)
 
 
+def test_save_nrrd_load_nrrd_str_Path():
```
Comment (GianlucaFicarelli) [inline tests/test_voxel_data.py:104]: I would split the test into 2 tests named `test_save_load_nrrd_as_path` and `test_save_load_nrrd_as_str`.

File: voxcell/voxel_data.py (hunk 1 above)
Comment (eleftherioszisis) [reply voxcell/voxel_data.py:131]: Updated the docstring.

File: tests/test_voxel_data.py (hunk 2 above)
Comment (eleftherioszisis) [reply tests/test_voxel_data.py]: Fixed.

File: tests/test_voxel_data.py (hunk 3 above)
Comment (eleftherioszisis) [reply tests/test_voxel_data.py:124]: Done.

File: voxcell/voxel_data.py (hunk 1 above)
Comment (eleftherioszisis) [reply voxcell/voxel_data.py:131]: And made casting to string explicit for load_nrrd too.

**Review by GianlucaFicarelli — APPROVED** [review]

**Pull request merged**